website_multi_theme/models/website.py

```diff
@@ -170,6 +175,21 @@ def _multi_theme_activate(self):
                         copied_view.inherit_id = custom_layout
                         data.attrib["inherit_id"] = custom_layout.key
                     copied_view.arch = etree.tostring(data)
+                elif (copied_view.inherit_id and
```

Comment:
This is very hard to read. Can you somehow split this to be more readable (an `else`, then prepare variables and make the comparison).

Reply:
This should be better https://github.com/yelizariev/website/commit/52eff0a3d660b8776712582ce054036cd500215e